lib/OX/Admin/UI/SessionStorage.php

```diff
@@ -23,7 +23,9 @@ class OX_Admin_UI_SessionStorage implements OX_Admin_UI_Storage
     function __construct($id = null, $path = null)
     {
         if ('cli' !== PHP_SAPI && 'files' === ini_get('session.save_handler')) {
-            $path = session_save_path();
+            // Gotta make sure we get only the actual path, respecting PHP's M and N options, see:
+            // http://php.net/manual/en/session.configuration.php#ini.session.save-path
+            $path = end(explode(';',session_save_path()));
```

Comment:
This will raise a notice "Only variables should be passed by reference". Although hidden, we still try to write code that doesn't raise any notice or warning. Please use a temporary variable to store the array instead.